src/PublicApiAnalyzers/Core/CodeFixes/DeclarePublicApiFix.cs

```diff
@@ -51,16 +51,6 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
             return Task.CompletedTask;
         }
 
-        internal static TextDocument? GetUnshippedDocument(Project project)
-        {
-            return project.AdditionalDocuments.FirstOrDefault(doc => doc.Name.Equals(DeclarePublicApiAnalyzer.UnshippedFileName, StringComparison.Ordinal));
-        }
-
-        internal static TextDocument? GetShippedDocument(Project project)
-        {
-            return project.AdditionalDocuments.FirstOrDefault(doc => doc.Name.Equals(DeclarePublicApiAnalyzer.ShippedFileName, StringComparison.Ordinal));
-        }
```

Comment:
These were used by different code fixes. So I moved them to the new helper class.